src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -73,7 +73,7 @@ internal static IArgument CreateArgumentOperation(ArgumentKind kind, IParameterS
             //      Note this check doesn't cover all scenarios. For example, when a parameter is a generic type but the type of the type argument 
             //      is undefined.
             if ((object)optionalParametersMethod == null 
-                || boundNode.HasAnyErrors
+                || hasErrors
```

Comment:
I'll change it to HasAnyErrors at all call sites, it was not an intentional change.